owncloud/owncloud.py

```diff
@@ -66,19 +201,6 @@ def __str__(self):
                (self.share_id, self.target_file, self.link, self.token)
 
 
-class UserShare():
-    """User share information"""
-
-    def __init__(self, share_id, share, perms):
-        self.share_id = share_id
-        self.share = share
-        self.perms = perms
-
-    def __str__(self):
-        return "UserShare(id=%i,path='%s',perms=%s)" % \
-               (self.share_id, self.share, self.perms)
-
-
 class GroupShare():
```

Comment:
Remove GroupShare too ?

For some reason I don't see its usage on your branch, but I see it here https://github.com/owncloud/pyocclient/blob/master/owncloud/owncloud.py#L1212
You might need to rebase onto master to get this recently merged functionality